circle.yml

```diff
@@ -1,6 +1,27 @@
 machine:
   node:
     version: 6.2.1
+  environment:
```

Comment:
Is to avoid the use for npm. It still need an npm install to install yarn as a dev dependency. So we loose the interest of yarn.